internal/pkg/deploy/cloudformation/stack/testdata/environments/template-with-defaultvpc-flowlogs.yml

```diff
@@ -1003,14 +1003,20 @@ Resources:
       DomainName: !Ref AppDNSName
       PublicAccessDNS: !GetAtt PublicLoadBalancer.DNSName
       PublicAccessHostedZone: !GetAtt PublicLoadBalancer.CanonicalHostedZoneID
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Metadata:
+      'aws:copilot:description': 'A  log group to create a flow log for the VPC'
+    Properties:
+      RetentionInDays: 14
   FlowLog:
     Metadata:
       'aws:copilot:description': 'A flow log for the VPC to capture information about the IP traffic'
     Type: AWS::EC2::FlowLog
     Properties:
       DeliverLogsPermissionArn: !GetAtt FlowLogRole.Arn
       LogDestinationType: cloud-watch-logs
-      LogGroupName: !Join ['-', [!Ref AppName, !Ref EnvironmentName, FlowLogGroup]]
+      LogGroupName: !Ref LogGroup
```

Comment:
I think this will be backwards incompatible for users that have already enabled flow logs using the environment manifest. The update requires a replacement: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-flowlog.html#cfn-ec2-flowlog-loggroupname

Can you test the following:
*  What happens after your change to the previous log group? is it deleted or retained. 
   * If retained, I think this is okay.
   * If not, then I wonder if there is a way of using the same log group name as before but changing it's retention.

Reply:
1. After changing previous log group name, it is still retained.
2. A new log group is created after the change to previous log group name.

![Screen Shot 2022-11-09 at 3 08 17 PM](https://user-images.githubusercontent.com/71282729/200961872-9c72c433-1779-44de-887e-7668c4bc94d6.png)

Reply:
Now the log group name is going to be same as before `AppName-EnvironmentName-FlowLogGroup`

Reply:
If it's the same name and a user had previously created the log group with:
```yaml
network:
 vpc:
   flow_logs: on
```
What happens?

 I assume CFN would fail saying the log group already exists 🤔 Would you mind testing that? 
 
 If it doesn't fail, perfect then retention was added successfully I assume?
 If it fails, we can just name the new log group `AppName-EnvironmentName-FlowLogs`

Reply:
Oh yes , CFN fails saying that the log group already exists. So I created a new log group with the suggested name `AppName-EnvironmentName-FlowLogs`.

With this new log group creation we will have both log groups 
`AppName-EnvironmentName-FlowLogGroup` 
`AppName-EnvironmentName-FlowLogs`